svg/extensibility/interfaces/foreignObject-graphics.svg

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg xmlns="http://www.w3.org/2000/svg"
+     xmlns:h="http://www.w3.org/1999/xhtml">
+  <metadata>
+    <link xmlns="http://www.w3.org/1999/xhtml" rel="help" href="https://www.w3.org/TR/2013/WD-SVG2-20130618/extend.html#ForeignObjectElement"/>
+    <link xmlns="http://www.w3.org/1999/xhtml" rel="help" href="https://svgwg.org/svg2-draft/changes.html#extend"/>
+    <meta xmlns="http://www.w3.org/1999/xhtml" name="assert" content="foreignObject is a graphics element"/>
+  </metadata>
+  <foreignObject id="target" />
+  <h:script src="/resources/testharness.js"/>
+  <h:script src="/resources/testharnessreport.js"/>
+  <h:script><![CDATA[
+  test(function() {
+    var target = document.getElementById('target');
+    assert_true(target instanceof SVGGraphicsElement);
```

Comment:
`instanceof` is defined to walk the entire prototype chain; we should be able to assert something more specific as to where exactly it is in the prototype chain.

I think what we want is `assert_equals(target.__proto__.__proto__, SVGGraphicsElement.prototype)`?

That said, this is essentially just testing `interface SVGForeignObjectElement : SVGGraphicsElement`, i.e., what interface `SVGForeignObjectElement` inherits from I believe? That, at least, will be tested by `idlharness.js` in /svg/interfaces.html, specifically by the test it creates with the name of "SVGForeignObjectElement interface: existence and properties of interface object".